Adds support to phoenix 1.6 and removes support to phoenix 1.3.

gabrielpedepera opened the pull request:
#9 

The phoenix 1.4 was released on `Nov 07, 2018` as a new version from the `1.3` and I'm looking to have `simple_auth` available for phoenix `1.6`.

https://hex.pm/packages/phoenix/versions

chazsconi commented on mix.exs:
```diff
@@ -38,7 +38,7 @@ defmodule SimpleAuth.Mixfile do
   defp deps do
     [
-      {:phoenix, "~> 1.3.0 or ~> 1.4.0 or ~> 1.5.0"},
+      {:phoenix, "1.4.0 or ~> 1.5.0 or ~> 1.6.0"},
```
I think there should be a `~` before `1.4.0`

gabrielpedepera replied:
Indeed. 😅

chazsconi commented on CHANGELOG.md:
```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+## v1.10.2
```
I think the next version should be `1.11.0` as this is a breaking change, however you can remove this line and just add the list of things that is unreleased and I'll add the version when I release it.

gabrielpedepera replied:
You're right !! I also dropped the line. 😉

chazsconi commented on mix.exs:
```diff
@@ -4,7 +4,7 @@ defmodule SimpleAuth.Mixfile do
     [
       app: :simple_auth,
-      version: "1.10.1",
+      version: "1.10.2",
```
Same as above

chazsconi reviewed (changes requested):
Thanks a lot for the PR!  If you can fix this comments I'll merge and release.

Commit:
Add supports to phoenix 1.6 and remove supports to phoenix 1.3

gabrielpedepera commented:
> Thanks a lot for the PR! If you can fix this comments I'll merge and release.

Thank you @chazsconi . I have addressed the comments, but anything else that perhaps I missed. Just let me know. 🙇‍♂️

chazsconi commented:
Thanks a lot @gabrielpedepera !
